Confus/ConfusServer/Game.cpp

```diff
@@ -154,7 +148,10 @@ namespace ConfusServer
 
     void Game::handleInput()
     {
-        m_PlayerNode.handleInput(m_EventManager);
+        for(auto player : m_PlayerArray)
+        {
+            player->handleInput(m_EventManager);
+        }
```

Comment:
You're right (as always) We shouldn't be checking on the server for input, so I removed it.